app/src/server/api/external/v1Api.router.ts

```diff
@@ -334,6 +301,149 @@ export const v1ApiRouter = createOpenApiRouter({
 
       return { status: "ok" };
     }),
+  updateLogTags: openApiProtectedProc
+    .meta({
+      openapi: {
+        method: "POST",
+        path: "/update-log-tags",
+        description: "Update tags for logged calls matching the provided filters",
+        protect: true,
+      },
+    })
+    .input(
+      z.object({
+        filters: z
+          .union([
+            z.object({
+              field: z.union([z.enum(["model", "completionId"]), z.string()]),
+              equals: z.union([z.string(), z.number(), z.boolean()]),
```

Comment:
If we're only matching the "model" and "completionId" field shouldn't the allowed value only be of type `string`?

Reply:
We also match tags.